README.md

```diff
@@ -1,30 +1,34 @@
 # Simple generator for Kubernetes resources
 
-This generator creates the Kubernetes `service` and `deploymnet` resources files to deploy a Spring Boot application using `kubectl`
-
-You will need to install the following command line tool:
+If you used the starter service, then the generator has already been executed and you can build and run the application and provision it's dependent services.
 
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
+### Deploy required services to Kubernetes
 
-## Generator Commands
+If using `mysql` service, create a secret
 
-`k8s-simple new` creates
+```
+kubectl create secret generic mysql \
+  --from-literal=mysql-root-password=$(echo $RANDOM) \
+  --from-literal=mysql-password=$(echo $RANDOM)
+```
 
-* A `kubernetes` directory with a `service.yaml` and `deployment.yaml`
+If you recreate the secret and have already run the `mysql` service, also delete the persistent-volume in addition to the secret
 
+```
+kubectl delete secret mysql
+kubectl delete pvc mysql
+```
 
-## Generator installation
+Now deploy the services:
 
 ```
-tss generator install --go-getter-url=github.com/markpollack/generator-k8s-simple
+kubectl apply -f kubernetes/services/**
```

Comment:
I think it's clearer to use the fully qualified `services/mysql` since there's just one service for now.